README.md

```diff
@@ -0,0 +1,75 @@
+# Comic Dance Hall
+This is an Dance hall webapp that allows a user to sign up, log in, and make leases on particular apartments. The user, apartments, and leases are stored in this API.
+
+
+## Built with
+
+- HTML
+- CSS
+- React
+- Redux
+- JavaScript
+- VScode
+- Lots of love :heart:
+
+
+## Previews
+
+
+
+
+## Live Demo
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Setup
+
+- Clone this repository. 
+- Run ``npm i`` or ``yarn`` to install dependencies required to start the project
+- Using node v15.11.0 (npm v7.6.0) 
+- Run ``npm start`` 
+
+## Run Tests
+
+```
+npm run lint
+```
+
+## Author:
+
+👤 **Moise Hernandez**
+
+[<code><img height="26" src="https://cdn.iconscout.com/icon/free/png-256/github-153-675523.png"></code>](https://github.com/alexander16108)
+
+
+[<code><img height="26" src="https://upload.wikimedia.org/wikipedia/sco/thumb/9/9f/Twitter_bird_logo_2012.svg/1200px-Twitter_bird_logo_2012.svg.png"></code>](https://linkedIn.com/codingrex)
+
+
+👤 **Moise Hernandez**
+
+[<code><img height="26" src="https://cdn.iconscout.com/icon/free/png-256/github-153-675523.png"></code>](https://github.com/mhdez6423)
+[<code><img height="26" src="https://upload.wikimedia.org/wikipedia/sco/thumb/9/9f/Twitter_bird_logo_2012.svg/1200px-Twitter_bird_logo_2012.svg.png"></code>](https://twitter.com/mhdez6423)
```

Comment:
Please add yoursefl as an author

you can use the one I use on the rails setup

-------------

## Authors

👤 **Moises Hernandez**

- GitHub: [@Mhdez221993](https://github.com/Mhdez221993)
- Twitter: [@MoisesH42060050](https://twitter.com/MoisesH42060050)
- LinkedIn: [Moises Hernandez Coronado](https://www.linkedin.com/in/moises-hernandez-9bbb17145/)

👤 **Alexander Odufuye**

- GitHub: [@alexander16108](https://github.com/alexander16108)
- LinkedIn: [alexander](https://www.linkedin.com/in/codingrex/)